--- vdr-plugin-mp3-0.10.4/mp3.c.orig	2024-09-30 17:31:13.649226101 +0200
+++ vdr-plugin-mp3-0.10.4/mp3.c	2024-09-30 17:34:25.297648813 +0200
@@ -281,7 +281,12 @@
 bool cMP3Control::SetPlayList(cPlayList *plist)
 {
   bool res;
-  cControl *control=cControl::Control();
+#if APIVERSNUM >= 20402
+  cMutexLock mtx;
+  cControl *control=cControl::Control(mtx);
+#else
+   cControl *control=cControl::Control();
+#endif
   // is there a running MP3 player?
   if(control && typeid(*control)==typeid(cMP3Control)) {
     // add songs to running playlist
@@ -1694,7 +1699,12 @@
     else { ReplyCode=501; return "Missing filename"; }
     }
   else if(!strcasecmp(Command,"CURR")) {
-    cControl *control=cControl::Control();
+#if APIVERSNUM >= 20402
+  cMutexLock mtx;
+  cControl *control=cControl::Control(mtx);
+#else
+   cControl *control=cControl::Control();
+#endif
     if(control && typeid(*control)==typeid(cMP3Control)) {
       cMP3PlayInfo mode;
       if(mgr->Info(-1,&mode)) return mode.Filename;
