diff -up mp3-0.10.1/config.h~ mp3-0.10.1/config.h
--- mp3-0.10.1/config.h~	2005-11-26 14:37:14.000000000 +0200
+++ mp3-0.10.1/config.h	2008-03-16 13:28:37.000000000 +0200
@@ -32,8 +32,8 @@
 
 // Uncomment to enable generic debugging messages to the console. This may slow
 // down operation in some cases.
-#define DEBUG
-//#define NO_DEBUG
+//#define DEBUG
+#define NO_DEBUG
 
 // Defines the filename extention to use for playlist files.
 #define PLAYLISTEXT ".m3u"
